doc/workflow.md

```diff
@@ -0,0 +1,163 @@
+# Tracing CVs with Inkscape from publication for the echemdb.website
+
+The cyclic voltammogram we trace is provided in Figure 2a in the [publication example](publication_example.pdf) and looks as follows:
+
+[<img src="sample_data_2.png" width="450"/>](sample_data_2.png)
+
+## Step 1: pdf, bib and metadata file preparation
+
+**1: Create a folder with the pdf, where the folder name is:**
+
+`FirstAuthorName_Year_FirstTitleWord_FirstPageNr`
+
+For the example pdf this comes down to
+
+`mustermann_2021_svgdigitizer_1`
+
+The Page number should be the page number in the published pdf. Since the example is not published the page number is `1`. In other cases this could be `1021`.
+
+The pdf should be named according to the same scheme:
+
+`mustermann_2021_svgdigitizer_1.pdf`
+
+**2: Place a bib file in the folder.**
+
+The most simple approach is to search the article with [Google Scholar](http://scholar.google.com/).
+
+Modify the scholar settings that a bibtex link appears below the citation:
+
+Click on the 3 lines next to the Google Scholar logo
+
+![./files/images/scholar_options.png](files/images/scholar_options.png)
+
+Choose Settings
+
+![./files/images/scholar_options_selection.png](files/images/scholar_options_selection.png)
+
+Select `Show links to import citatons into BibTex"
+
+![./files/images/scholar_options_bibtex.png](files/images/scholar_options_bibtex.png)
+
+An `import into BbTex` link appears below the linked article:
+
+![./files/images/scholar_options_bibtex_link.png](files/images/scholar_options_bibtex_link.png)
+
+Download the bib file or save the content to a file named:
+
+`mustermann_2021_svgdigitizer_1.bib`
+
+Open the file and change the key, such that it matches the folder name:
+
+![./files/images/bibtex_key.png](./files/images/bibtex_key.png)  
+
+
+
+**Finally the folder should now contain the following files:**
+
+`mustermann_2021_svgdigitizer_1.bib`
+
+`mustermann_2021_svgdigitizer_1.pdf`
+
+
+
+## Step 2: Prepare svg and png files from the pdf
+
+Install the `svgdigitizer`.
```

Comment:
We should link to some installation instructions here. If we do not have any, we should probably add a TODO + Issue for this.